src/services/item/test/index.test.ts

```diff
@@ -1927,6 +1945,35 @@ describe('Item routes tests', () => {
         expect(response.statusCode).toBe(StatusCodes.OK);
       });
 
+      it('Update successfully description placement above', async () => {
+        const { item } = await saveItemAndMembership({
+          member: actor,
+        });
+        const payload = {
+          settings: {
+            ...item.settings,
+            descriptionPlacement: DescriptionPlacement.ABOVE,
+          },
+        };
+
+        const response = await app.inject({
+          method: HttpMethod.Patch,
+          url: `/items/${item.id}`,
+          payload,
+        });
+
+        const newItem = response.json();
+
+        // this test a bit how we deal with extra: it replaces existing keys
+        expectItem(newItem, {
+          ...item,
+          ...payload,
+        });
+        expect(response.statusCode).toBe(StatusCodes.OK);
+        expect(newItem.settings.descriptionPlacement).toBe(DescriptionPlacement.ABOVE);
+        expect(newItem.settings.hasThumbnail).toBe(false);
+      });
+
```

Comment:
Add a test that throws for updating a setting that is not allowed? Since you nicely added this check in the schema!

Reply:
As mentioned in Slack, the schema will only filter the bad settings without throwing. For now, I written some tests to check this behaviour. If we want to throw an error, we will have to update the code.